src/main/drivers/light_ws2811strip.c

```diff
@@ -109,12 +110,18 @@ static int16_t ledIndex;
 #define USE_FAST_DMA_BUFFER_IMPL
 #ifdef USE_FAST_DMA_BUFFER_IMPL
 
-STATIC_UNIT_TESTED void fastUpdateLEDDMABuffer(rgbColor24bpp_t *color)
+STATIC_UNIT_TESTED void fastUpdateLEDDMABuffer(bool packedRGB, rgbColor24bpp_t *color)
```

Comment:
Can you add an enum like `outputFormat` and use it here and in other places where `packedRGB` is used? This will increase readability a bunch.